Assertion rewriting fails on OS X El Capitan

```
______________________________________________________________________________ ERROR collecting src/s3am/test/s3am_tests.py _______________________________________________________________________________
.eggs/py-1.4.30-py2.7.egg/py/_path/local.py:650: in pyimport
    __import__(modname)
src/s3am/__init__.py:15: in <module>
    import logging
/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/logging/__init__.py:26: in <module>
    import sys, os, time, cStringIO, traceback, warnings, weakref, collections
.eggs/pytest-2.7.2-py2.7.egg/_pytest/assertion/rewrite.py:113: in find_module
    os.mkdir(cache_dir)
E   OSError: [Errno 1] Operation not permitted: '/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/logging/__pycache__'
======================================================================================== 17 error in 1.22 seconds =========================================================================================
```

I believe SIP (https://en.wikipedia.org/wiki/System_Integrity_Protection) to be the cause.

--- a/_pytest/assertion/rewrite.py
+++ b/_pytest/assertion/rewrite.py
@@ -128,7 +128,7 @@ class AssertionRewritingHook(object):
                     # One of the path components was not a directory, likely
                     # because we're in a zip file.
                     write = False
-                elif e in [errno.EACCES, errno.EROFS]:
+                elif e in [errno.EACCES, errno.EROFS, errno.EPERM]:
                     state.trace("read only directory: %r" % fn_pypath.dirname)
                     write = False
                 else:

